### DISCUSSION TITLE
How to add to / set follow group from code?

### DISCUSSION THREAD

**Discussion by mghicks** [ramokz/phantom-camera#137] [Question / Help Wanted] [answered]
In the event I have a dynamic group of nodes I want followed, can I setup the follow group in code?  An example of this would be very helpful. The indirection through Properties makes it difficult to understand (for me, at least).

**Comment by ramokz** [comment, marked as answer]
You can indeed!

It's possible to do it either for an individual node or for an array of nodes via setter method.

```gdscript
# For individual nodes
pcam.append_follow_group_node(node)

# For an array of nodes
pcam.append_follow_group_node_array(node_array)
```

The documentation, with additional setter methods, can be found on [this page](https://github.com/ramokz/phantom-camera/wiki/Properties:-Follow-(2D-&-3D)#void-append_follow_group_node-node2d-node).
